client/src/components/QA/AnswersList.jsx

```diff
@@ -6,43 +6,68 @@ import DisplayPhotos from './DisplayPhotos';
 class AnswersList extends React.Component {
   constructor(props) {
     super(props);
+    this.answersLength = Object.keys(this.props.answers).length;
     this.state = {
-      answersDisplay: 0,
-      showButton: true
+      answersDisplay: 2,
+      showButton: false,
+      buttonLabel: 'Collapse answers'
     };
   }
 
   componentDidMount() {
-    const questionId = this.props.questionId;
-    this.checkRemainAnswers();
-  }
-
-  checkRemainAnswers() {
-    const answersLength = Object.keys(this.props.answers).length;
-    if (answersLength - 2 > this.state.answersDisplay) {
+    if (this.answersLength > this.state.answersDisplay) {
       this.setState({
         showButton: true,
-        answersLength: answersLength,
-        answersDisplay: this.state.answersDisplay + 2
+        buttonLabel: 'Load more answers'
       });
     }
-    if (answersLength - 2 <= this.state.answersDisplay) {
+  }
+
+  loadMoreAnswers() {
+    const { answersDisplay } = this.state;
+    if (answersDisplay === 2) {
       this.setState({
-        showButton: false,
-        answersLength: answersLength,
-        answersDisplay: this.state.answersDisplay + 2
+        answersDisplay: this.answersLength,
+        buttonLabel: 'Collapse answers'
+      });
+    } else {
+      this.setState({
+        answersDisplay: 2,
+        buttonLabel: 'Load more answers'
       });
     }
   }
 
-  render() {
+  sortAnswers() {
     const answers = this.props.answers;
-    const answersLength = Object.keys(answers).length;
+    let sellerAnswers = [];
+    let otherAnswers = [];
+    Object.keys(answers).forEach(answerId => {
+      if (answers[answerId].answerer_name === 'Seller') {
+        sellerAnswers.push(answers[answerId]);
+      } else {
+        otherAnswers.push(answers[answerId]);
+      }
+    });
+
+    sellerAnswers.sort(function (a, b) {
+      return b.helpfulness - a.helpfulness;
+    });
+    otherAnswers.sort(function (a, b) {
+      return b.helpfulness - a.helpfulness;
+    });
+
+    const result = sellerAnswers.concat(otherAnswers);
+    return result;
+  }
+
+  render() {
+    const answers = this.sortAnswers();
     const moreAnswersButton = () => {
       if (this.state.showButton) {
         return (
-          <div>
-            < button onClick={this.checkRemainAnswers.bind(this)} > more answers</button >
+          <div className='more_answer_button'>
+            < button onClick={this.loadMoreAnswers.bind(this)} >{this.state.buttonLabel}</button >
```

Comment:
Avoid binding functions in your render method. In your constructor: `this.loadMoreAnswers = this.loadMoreAnswers.bind(this);` In `onClick`: `this.loadMoreAnswers`